src/asmcnc/skavaUI/screen_go.py

```diff
@@ -671,7 +667,8 @@ def reset_go_screen_prior_to_job_start(self):
         self.sm.get_screen('home').z_datum_reminder_flag = False
 
         # Reset YP toggle
-        self.yp_widget.disable_yeti_pilot()
+        if not self.m.has_spindle_health_check_passed():
```

Comment:
will this remember whether the spindle health check passed for the previous job though?